### ISSUE TITLE
Products: No error when I update a product with no title

### ISSUE THREAD

**Issue by rachelmcr** [woocommerce/woocommerce-android#2055]
**Describe the bug**

I shouldn't be able to save a product with no title. If I delete the title when I'm editing the product details, I should get an error / should not be allowed to update the product.

**To Reproduce**

1. Go to the Products tab.
2. Select a product.
3. Delete the product title.
4. Tap "Update."

Notice that the product is updated and in the product list it is called "Untitled."

**Screenshots**

<img src="https://user-images.githubusercontent.com/8658164/76642295-d6907c80-654a-11ea-8adf-bae016ad9e8a.png" width="50%">

**Mobile Environment**
Please include:
- Device: moto e5 play
- Android version: 8.1.0
- WooCommerce Android version: 3.8-rc-1

**Comment by anitaa1990** [comment]
Thanks so much for testing @rachelmcr 🙏 

> No error when I update a product with no title

The reason we are able to update a blank product title is because it is supported in Core as well. The idea was to replicate the behaviour that is currently supported in `wp-admin`.  I'm not entirely certain why someone would try to update a blank product title for a published product though and why it is supposed in Core 😅

**Comment by rachelmcr** [comment]
Ah interesting! 🤔  I noticed it because updating to a blank product title is blocked with an error message on iOS right now. I also noticed that in the designs, the plan is to show an error if someone tries to create a new product without a title, but maybe that's a sufficiently different scenario?

I don't think this is a huge deal but it might be worth agreeing on the desired behavior for cross-platform consistency. cc @Garance91540
